.github/workflows/tests.yml

```diff
@@ -159,12 +162,14 @@ jobs:
     needs: syntax
     if: startsWith(github.repository, 'Homebrew/')
     runs-on: macos-13
-    env:
-      HOMEBREW_NO_INSTALL_FROM_API: 1
     steps:
       - name: Set up Homebrew
         id: set-up-homebrew
         uses: Homebrew/actions/setup-homebrew@master
+        with:
+          core: true
```

Comment:
Fine keeping this one given it already had `HOMEBREW_NO_INSTALL_FROM_API`, though not sure why cask audits need core?